generators/server/templates/src/main/java/package/config/MethodSecurityConfiguration.java.ejs

```diff
@@ -19,16 +19,22 @@
 package <%= packageName %>.config;
 
 import org.springframework.context.annotation.Configuration;
+<%_ if (authenticationType !== 'oauth2') { _%>
```

Comment:
as this file template is skipped in the oauth2 flow, do you still require these conditionals?